### ISSUE TITLE
Block download unbelievably slow

### ISSUE THREAD

**Issue by LeperKitty** [sero-cash/wallet#9]
Hi there. I downloaded your SERO wallet and transferred some SERO to it. It's currently downloading blocks so slowly that I estimate it will be about 4 years before I will have completed the download. The SERO have not yet appeared in the wallet after more than a month. Any advice?

Thanks!

**Comment by WagnerFlorian** [comment]
Hello, I have exactly the same issue.  Is there already a solution for this problem? Thanks!

**Comment by gabrieldan69** [comment]
please fix this issue ... i am waiting for 6 days and still at ~ 38%

**Comment by useer789** [comment]
try this maybe https://wiki.sero.cash/en/index.html?file=Start/sero-chain-snapshot-list

**Comment by fish101022** [comment]
Exactly the same, I have one PC working over a week now and still only at about 50% to download the blockchain, it's quite ridiculous. Don't you have a bootstrap file to download from here that gets us to around 90-95%?

**Comment by gabrieldan69** [comment]
Hello friends 

Here is the snapshot 🗡️ 
https://wiki.sero.cash/en/index.html?file=Start/sero-chain-snapshot-list

**Comment by mkcontr** [comment]
similar time to download it.. maybe anyone can put it on torrent network ?

> Hello friends
> 
> Here is the snapshot dagger https://wiki.sero.cash/en/index.html?file=Start/sero-chain-snapshot-list

**Comment by fish101022** [comment]
> similar time to download it.. maybe anyone can put it on torrent network ?
> 
> > Hello friends
> > Here is the snapshot dagger https://wiki.sero.cash/en/index.html?file=Start/sero-chain-snapshot-list

Agree, the site will only allow you to download 5gb per 6 hours unless you pay about US$5. (Yes, it goes by IP address so just VPN and continue but it's annoying)... But most importantly that most recent snapshot is still missing nearly 2 million blocks from where Sero chain is at today, so once that's downloaded and placed into the folder, we're still looking at another week to finish syncing. Something is very wrong there, other blockchains only take a day or two to sync in full. 

On the Sero Wallet, it says I'm connected to 15 peers, but it's crazy slow. If that were a torrent site with 15 peers connected, a 150gb file would take a couple hours at most.

**Comment by mkcontr** [comment]
> maybe anyone can put it on torrent network ?

anyone?

EDIT: asking, because I'm still syncing..

**Comment by dragonfly62** [comment]
I am having the same issue ;  and it is impossible to contact any support ... it really looks like a poorly managed project to me

**Comment by gabrieldan69** [comment]
the support is 0 !!!  i quit this  shitcoin
